Installation/cmake/modules/FindQGLViewer.cmake

```diff
@@ -69,6 +77,41 @@ IF(QGLVIEWER_INCLUDE_DIR AND QGLVIEWER_LIBRARIES)
    SET(QGLVIEWER_FOUND TRUE)
 ENDIF(QGLVIEWER_INCLUDE_DIR AND QGLVIEWER_LIBRARIES)
 
+if(Qt5_FOUND AND OpenGL_FOUND AND QGLVIEWER_FOUND)
+  if(NOT TARGET QGLViewer::QGLViewer)
+    add_library(QGLViewer::QGLViewer UNKNOWN IMPORTED)
+
+    set_target_properties(QGLViewer::QGLViewer PROPERTIES
+      INTERFACE_COMPILE_DEFINITIONS "${QGLVIEWER_DEFINITIONS}"
+      INTERFACE_INCLUDE_DIRECTORIES "${OPENGL_INCLUDE_DIR}"
+      INTERFACE_LINK_LIBRARIES Qt5::Xml
+      )
+
+    set_property(TARGET QGLViewer::QGLViewer APPEND PROPERTY INTERFACE_INCLUDE_DIRECTORIES
+      "${QGLVIEWER_INCLUDE_DIR}")
+    
+    set_property(TARGET QGLViewer::QGLViewer APPEND PROPERTY INTERFACE_LINK_LIBRARIES
+                           Qt5::OpenGL)
+    set_property(TARGET QGLViewer::QGLViewer APPEND PROPERTY INTERFACE_LINK_LIBRARIES
+                           "${OPENGL_LIBRARIES}")
```

Comment:
With certain platforms (and certainly on Windows), `${OPENGL_LIBRARIES}` will have `debug` and `optimized` keyword in it, like in `debug;libGL.so;release;libGLd.so`, and that call will not work.

Could not you use instead:
```cmake
target_link_libraries(QGLViewer::QGLViewer INTERFACE "${OPENGL_LIBRARIES}")
```
instead of filling the `INTERFACE_LINK_LIBRARIES` property by hand?

Reply:
No it is not possible to use target_link_libraries for an imported target (cf [here](https://cmake.org/cmake/help/v3.0/manual/cmake-buildsystem.7.html#imported-targets))

Reply:
Actually, I read the `FindOpenGL.cmake` module from both CMake 3.9 and CMake 3.1, and none of them use `debug` and `optimized` in their output variable `OPENGL_LIBRARIES`. So let's ignore that non-issue.

I agree with the patch, now.